### PR TITLE
Update embedded appearance API and add to playground

### PR DESCRIPTION
**Pull request by porter-stripe**
## Summary
- Updates the embedded appearance API based on the outcome of API review
- Updates the playground appearance view to have the new properties from the embedded appearance API

https://github.com/user-attachments/assets/4b802d3f-6ec6-45d2-af89-a9891e77e177

## Motivation
- Embedded

## Testing
- Manual

## Changelog
N/A

### CODE REVIEW

File: Example/PaymentSheet Example/PaymentSheet Example/AppearancePlaygroundView.swift [hunk 1]
```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2022 stripe-ios. All rights reserved.
 //
 
-@_spi(STP) import StripePaymentSheet
+@_spi(STP) @_spi(EmbeddedPaymentElementPrivateBeta) import StripePaymentSheet
```
Comment (yuki-stripe) [inline Example/PaymentSheet Example/PaymentSheet Example/AppearancePlaygroundView.swift:9]: Can we remove (STP)?  Leaving it risks not noticing if we break visibility of an API.

File: Example/PaymentSheet Example/PaymentSheet Example/AppearancePlaygroundView.swift [hunk 2]
```diff
@@ -237,6 +284,7 @@ struct AppearancePlaygroundView: View {
                 Section(header: Text("Miscellaneous")) {
                     Stepper(String(format: "cornerRadius: %.1f", appearance.cornerRadius), value: cornerRadiusBinding, in: 0...30)
                     Stepper(String(format: "borderWidth: %.1f", appearance.borderWidth), value: borderWidthBinding, in: 0.0...2.0, step: 0.5)
+                    Stepper(String(format: "selectedBorderWidth: %.1f", appearance.selectedBorderWidth ?? appearance.borderWidth * 1.5), value: borderWidthBinding, in: 0.0...2.0, step: 0.5)
```
Comment (yuki-stripe) [inline Example/PaymentSheet Example/PaymentSheet Example/AppearancePlaygroundView.swift:287]:
```suggestion
                    Stepper(String(format: "selectedBorderWidth: %.1f", appearance.selectedBorderWidth ?? appearance.borderWidth * 1.5), value: selectedBorderWidthBinding, in: 0.0...2.0, step: 0.5)
```
right?

File: StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetAppearance.swift [hunk 3]
```diff
@@ -234,54 +234,54 @@ public extension PaymentSheet {
...
 
-                /// The color of the separator line between payment method rows
+                /// The color of the separator line between rows
                 /// - Note: If `nil`, defaults to `appearance.colors.componentBorder`
                 public var separatorColor: UIColor?
 
-                /// The insets of the separator line between payment method rows
-                public var separatorInset: UIEdgeInsets?
+                /// The insets of the separator line between rows
+                public var separatorInsets: UIEdgeInsets?
```
Comment (yuki-stripe) [inline StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetAppearance.swift:279]: Let's add a note about behavior when nil.

**Review by yuki-stripe — COMMENTED** [review]
Awesome. BTW let's use the `embedded` label in PRs going forward!

File: Example/PaymentSheet Example/PaymentSheet Example/AppearancePlaygroundView.swift (hunk 2 above)
Comment (porter-stripe) [reply Example/PaymentSheet Example/PaymentSheet Example/AppearancePlaygroundView.swift]: ah good eye!

**Review by yuki-stripe — APPROVED** [review]

**Pull request merged**